pkgs/development/tools/dep/default.nix

```diff
@@ -21,7 +21,6 @@ buildGoPackage rec {
     homepage = https://github.com/golang/dep;
     description = "Go dependency management tool";
     license = licenses.bsd3;
-    platforms = platforms.linux;
```

Comment:
Can you use `platforms.all` instead? That way we don't rely on the default value being sensible.

Reply:
Sure, just a moment.